_posts/2019-07-19-paho-mqtt.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: post
+current: post
+cover: ethernet-cables.jpeg
+title: Using WoTT to Secure a Paho MQTT Client
+date: 2019-07-19 17:00:00
+category: tutorials
+author: Fiona McAllister
+tags: [Open Source]
+class: post-template
+---
+
+# Using WoTT to Secure a Connection from a Paho MQTT Client to an MQTT Server (Mosquitto MQTT Example included)
+
+## Introduction
+
+In a previous [example]({{site.url}}/blog/tutorials/2019/07/15/mosquitto-mqtt) we discussed how to secure connections between Mosquitto brokered MQTT client and server. In that there are several different brokers for an MQTT server, there are also several client implementations for the MQTT protocol. There are several libraries available for multiple MQTT clients in various programming languages. This example looks at Paho, another Eclipse run enterprise that specifically implements a client for MQTT. We will be looking at the Python implementation of Paho. Although you can use any other MQTT server, we already have a Mosquitto server from our previous example that we can use for this tutorial.
```

Comment:
'we discussed how to secure connections between Mosquitto brokered MQTT client and server' -> 'we discussed how to secure connections between an Mosquitto MQTT broker and the Mosquitto client''

Reply:
'several different brokers for an MQTT server' -> 'several different MQTT servers/brokers'

Reply:
'Eclipse run enterprise' -> 'project adopted into the Eclipse Foundation'